wscl-issues/draft/boolean-definition

```diff
@@ -0,0 +1,68 @@
+Issue:          BOOLEAN-DEFINITION
+Forum:          Cleanup
+Category:       CLARIFICATION
+Status:         draft
+Edit History:   16-May-24, Version 1 by Jin-Cheng Guu
+References:     BOOLEAN
+
+Problem Description:
+
+  In the draft ANSI Common Lisp specification, the description of the type
+  BOOLEAN [a] can be more accurate.
+
+    [a]: The type boolean contains the symbols t and nil, [..]
+
+  It is more accurate to use the word "consists of" than "contains". After
+  all, according to the glossary [b], BOOLEAN is exclusively either the symbol
+  T or the symbol NIL.
+
+    [b]: boolean n. an object of type boolean; that is, one of the following
+    objects: the symbol t (representing true), or the symbol nil (representing
+    false).
+
+Proposal:
+
+  This proposal changes the description of the type BOOLEAN so that [a]
+  instead reads:
+
+    > The type boolean consists of the symbols t and nil, [..]
+
+Test Cases:
+
+  N/A (Test cases are difficult to provide, because conforming implementations
+  are not expected to expose its definition of the type BOOLEAN
```

Comment:
s/its definition/their definitions/

Reply:
Grammar fixed. However, as we are adding a test as you suggested above, this part will be taken away later.